README.md

```diff
@@ -1,17 +1,31 @@
-# Sample
+﻿# About
+
+
 
 This module is for fun and trying out new crazy and wacky things with the game. It houses a number of example blocks and UI screens.
 
-### Credits for images:
+ 
+A person can test their UI screens and various other creations in this module.It is more focused towards developers who want to test stuff, but players can enjoy it too!Check out this module at https://github.com/Terasology/Sample
```

Comment:
Please remove the "Check this module out at" but and the link.

Reply:
Also, space after every punctuation. `..in this module.It is more..` ->`..in this module. It is more..`
Make this change for all the lines.